src/Menu.cpp

```diff
@@ -701,8 +703,18 @@ HMENU BuildMenu(WindowInfo *win)
         RebuildFavMenu(win, m);
         AppendMenu(mainMenu, MF_POPUP | MF_STRING, (UINT_PTR)m, _TR("F&avorites"));
     }
+
+    // Build the themes sub-menu of the settings menu
     m = BuildMenuFromMenuDef(menuDefSettings, dimof(menuDefSettings), CreateMenu(), filter);
+    MenuDef menuDefTheme[THEME_COUNT];
+    static_assert(IDM_CHANGE_THEME_LAST - IDM_CHANGE_THEME_FIRST + 1 >= THEME_COUNT, "Too many themes. Either remove some or update IDM_CHANGE_THEME_LAST");
+    for (UINT i = 0; i < THEME_COUNT; i++) {
+        menuDefTheme[i] = { _TRN(GetThemeByIndex(i)->name), IDM_CHANGE_THEME_FIRST + i, 0 };

```

Comment:
The way translation system works is:
* _TRN() marks a literal string that needs to be translated. Here you need to call `trans::GetTranslation(GetThemeByIndex....)`